bokeh/application/handlers/code.py

```diff
@@ -54,12 +55,8 @@ def modify_document(self, doc):
 
         module = self._runner.new_module()
 
-        # This is to prevent the module from being gc'd before the
-        # document is.  A symptom of a gc'd module is that its
-        # globals become None.
```

Comment:
Sure OK that's true, but it's a really important sub-reason :) I think it's worth noting, maybe:
```
# One reason modules are stored is to prevent the module 
# from being gc'd before the document is.  (A symptom of a 
# gc'd module is that its globals become None.) Additionally
# stored modules are used to provide correct paths. 

```